axlearn/common/input_ranking.py

```diff
@@ -0,0 +1,54 @@
+# Copyright © 2023 Apple Inc.
+
+"""Input processing utilities on tf.data for ranking-related tasks."""
+from typing import Dict
+
+import seqio
+import tensorflow as tf
+
+from axlearn.common import input_tf_data
+
+
+def rank_by_value(
+    *, input_key: str, output_key: str, ascending: bool = True, allow_ties: bool = False
```

Comment:
Nit: remove default values, as they are not obvious choices, so spelling them out explicitly at call sites will make the code more readable.

```suggestion
    *, input_key: str, output_key: str, ascending: bool, allow_ties: bool,
```

Reply:
Done.